test/query.test.js

```diff
@@ -1341,53 +1253,55 @@ describe('Query', function() {
         query = new Query({});
         query.tailable(false);
         assert.equal(query.options.tailable, false);
-        done();
       });
-      it('supports passing the `awaitData` option', function(done) {
+      it('supports passing the `awaitData` option', function() {
         const query = new Query({});
         query.tailable({ awaitData: true });
         assert.equal(query.options.tailable, true);
         assert.equal(query.options.awaitData, true);
-        done();
       });
     });
 
     describe('comment', function() {
-      it('works', function(done) {
+      it('works', function() {
         const query = new Query();
         assert.equal(typeof query.comment, 'function');
         assert.equal(query.comment('Lowpass is more fun'), query);
         assert.equal(query.options.comment, 'Lowpass is more fun');
-        done();
       });
     });
 
     describe('hint', function() {
-      it('works', function(done) {
+      it('works', function() {
         const query2 = new Query({});
         query2.hint({ indexAttributeA: 1, indexAttributeB: -1 });
         assert.deepEqual(query2.options.hint, { indexAttributeA: 1, indexAttributeB: -1 });
 
         const query3 = new Query({});
         query3.hint('indexAttributeA_1');
         assert.deepEqual(query3.options.hint, 'indexAttributeA_1');
+      });
+    });
 
-        done();
+    describe('snapshot', function() {
+      it('works', function() {
+        const query = new Query({});
+        query.snapshot(true);
+        assert.equal(query.options.snapshot, true);
       });
     });
```

Comment:
i think this was accidentally added back in the merge of 7.0 into this PR

weirdly the tests seem to pass though

Reply:
mquery still has a snapshot method, that's why this test still succeeds. I guess we should also remove these from mquery.